module-bsp/board/rt1051/bsp/lpm/RT1051LPMCommon.cpp

```diff
@@ -28,6 +28,7 @@ namespace bsp
 
     int32_t RT1051LPMCommon::PowerOff()
     {
+        SetCpuFrequency(bsp::CpuFrequencyMHz::Level_2);
```

Comment:
Shouldn't it be placed in the Bell specific board exit?

Reply:
well if we want this fix only for Harmony then okay :)

Reply:
From what I know, Pure does not use SNVS.

Reply:
Yes, Pure indeed doesn't use SNVS.